streams/src/test/java/org/apache/kafka/streams/kstream/MaterializedTest.java

```diff
@@ -32,23 +36,62 @@ public void shouldAllowValidTopicNamesAsStoreName() {
         Materialized.as("valid_name");
     }
 
-    @Test(expected = TopologyException.class)
+    @Test
     public void shouldNotAllowInvalidTopicNames() {
-        Materialized.as("not:valid");
+        final String invalidName = "not:valid";
+        final TopologyException e = assertThrows(TopologyException.class,
+            () -> Materialized.as(invalidName));
+
+        assert e.getMessage().equals("Invalid topology: Name \"" + invalidName +
```

Comment:
We should not use Java `assert` statement but proper unit testing asserts, ie, `assertThat(e.getMessage(), equalTo("..."));`

Reply:
Used junit assertEquals method instead of java assert

Reply:
Thanks. Will merge after Jenkins is green.